### PR TITLE
Donut Meta CIP Draft

### PR DESCRIPTION
**Pull request by YazzyYaz**
Initial Draft for the Donut Meta CIP Hardfork Specification Document

### CODE REVIEW

File: CIPs/cip.donut.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+This is the Meta CIP for the technical specifications of the Celo Donut Hardfork.
+
+## Simple Summary
+
+Enable the activation of the Celo Donut Hardfork specification which includes Ethereum Berlin EIPs and Celo CIPs.
+
+## Abstract
+
+Add support for protocol-impacting changes introduced in the Ethereum Berlin specification for Yolov3 and Celo improvement proposals in a hardfork
+named by the community as ____Donut_.
```
Comment (prestwich) [inline CIPs/cip.donut.md:21]: Got some extra underscores there

File: CIPs/cip.donut.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+The enhance the EVM's capabilities and while pushing forward the Celo blockchain functionality in order to enable economic conditions for prosperity as well as adopting Ethereum's Berlin specifications.
+
+## Specification
+The Donut Hardfork specification meta document includes the following proposals:
+* [EIP 665: Ed25519 Signature Verification](https://eips.ethereum.org/EIPS/eip-665)
+* [EIP 2537: BLS Curve Operations on 12-381](https://eips.ethereum.org/EIPS/eip-2537)
+* [EIP 2539: BLS Curve Operations on 12-377](https://eips.ethereum.org/EIPS/eip-2539)
+* [CIP 20: Extensible Hash Function Precompile](https://github.com/celo-org/celo-proposals/pull/44)
+* [CIP 21: Governable Lookback Window](https://github.com/celo-org/celo-proposals/pull/48)
+* [CIP 22: Epoch Snark Data](https://github.com/celo-org/celo-proposals/pull/67/files#diff-57a5e04aa05b40794aa8beb293d2b68c967dab4d7f625546baec17a8e5bb568b)
```
Comment (prestwich) [inline CIPs/cip.donut.md:43]: Is there an CIP for split etherbase? Also, how about the unnumbered CIP for retrieving BLS pubkeys?

File: CIPs/cip.donut.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+cip: 
+title: Donut Hardfork 
+author: Yaz Khoury <@YazzyYaz>
```
Comment (YazzyYaz) [inline CIPs/cip.donut.md:4]: must add James

File: CIPs/cip.donut.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+## Specification
+The Donut Hardfork specification meta document includes the following proposals:
+* [EIP 665: Ed25519 Signature Verification](https://eips.ethereum.org/EIPS/eip-665)
+* [EIP 2537: BLS Curve Operations on 12-381](https://eips.ethereum.org/EIPS/eip-2537)
+* [EIP 2539: BLS Curve Operations on 12-377](https://eips.ethereum.org/EIPS/eip-2539)
+* [CIP 20: Extensible Hash Function Precompile](https://github.com/celo-org/celo-proposals/pull/44)
+* [CIP 21: Governable Lookback Window](https://github.com/celo-org/celo-proposals/pull/48)
+* [CIP 22: Epoch Snark Data](https://github.com/celo-org/celo-proposals/pull/67/files#diff-57a5e04aa05b40794aa8beb293d2b68c967dab4d7f625546baec17a8e5bb568b)
+
+## Rationale
```
Comment (prestwich) [inline CIPs/cip.donut.md:45]: No longer applies as written. 

Suggestions for updates:
1. Functionality: We add a number of new features and precompiles to enable new cryptographic protocols and support Plumo.
2. Validator Quality of life: CIPs 21 and (split etherbase?) address common validator needs wrt reward allocation.

File: CIPs/cip.donut.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+For more information on the opcodes and their respective EIPs and CIP implementations, please see the _Specification_
+section of this document.
+
+## Motivation
+
+The enhance the EVM's capabilities and while pushing forward the Celo blockchain functionality in order to enable economic conditions for prosperity as well as adopting Ethereum's Berlin specifications.
+
+## Specification
+The Donut Hardfork specification meta document includes the following proposals:
+* [EIP 665: Ed25519 Signature Verification](https://eips.ethereum.org/EIPS/eip-665)
```
Comment (prestwich) [inline CIPs/cip.donut.md:38]: update to reference CIP25

**Review by prestwich — CHANGES_REQUESTED** [review]

File: CIPs/cip.donut.md (hunk 2 above)
Comment (YazzyYaz) [reply CIPs/cip.donut.md:44]: Let's add them in once we hear from some of the authors. Can merge this first version since it's still a first draft.

**Review by prestwich — APPROVED** [review]

**Pull request merged**